dm_control/locomotion/walkers/assets/dog_v2/dog.xml

```diff
@@ -1,12 +1,13 @@
 <mujoco model="dog">
   <compiler meshdir="../../../../suite/dog_assets" texturedir="../../../../suite/dog_assets"/>
 
-  <option timestep="0.005" noslip_iterations="4"/>
+  <option timestep="0.005" integrator="implicitfast" cone="elliptic" solver="Newton" iterations="1000" noslip_iterations="15"/>
```

Comment:
Newton solver is default, delete.
you don't need 1000 iterations, delete.
3 noslip iterations should be enough